stdlib/asm/mem.masm

```diff
@@ -141,7 +142,7 @@ export.pipe_words_to_memory.0
     # => [B', A, write_ptr+1, ...]
 
     # Push padding word (4 cycles)
-    push.1.0.0.0
+    push.0.0.0.0
```

Comment:
minor nit: we could use `padw` instead of `push.0.0.0.0`.

Reply:
Done